pages/product/index.vue

```diff
@@ -267,10 +268,10 @@ export default defineComponent({
 
   setup(_, context) {
     const { productCode } = context.root.$route.params
-    const { load, product, configure, loading, error } = useProductSSR(productCode)
+    const { load, product, configure, setFulfillment, loading, error } = useProductSSR(productCode)
     const { toggleStoreLocatorModal } = useUiState()
     const { purchaseLocation, load: loadPurchaseLocation } = usePurchaseLocation()
-    const selectedFulfillmentOption = ref()
+    const selectedFulfillmentValue = ref()
```

Comment:
Lets make this a computed property and add a getter for it
selectedFulfillmentValue = computed(() => productGetters.getFulfillmentMethod(product.value)